source/operations/checklists/hardware.rst

```diff
@@ -8,4 +8,233 @@ Hardware Checklist
+
+   * - :octicon:`circle`
+     - | Sufficient RAM to achieve performance goals based on the number of drives and anticipated concurrent requests (see the :ref:`formula and reference table <minio-requests-per-node>`)
+       | Refer to the information on :ref:`memory allocation <minio-k8s-production-considerations-memory>` for recommended RAM amounts 
+
+   * - :octicon:`circle`
+     - Four nodes or servers
+
+   * - :octicon:`circle`
+     - | SATA/SAS drives for capacity and NVMe SSDs for high-performance
+       | MinIO recommends a minimum of 8 drives per server
+
+   * - :octicon:`circle`
+     - | 25GbE network for capacity 
+       | 100GbE Network interface cards for high performance
+
+.. important:: 
+
+   The following areas have the greatest impact on MinIO performance, listed in order of importance:
+
+   - Network infrastructure (insufficient or limited throughput)
+   - Storage controller (old firmare, limited throughput)
+   - Storge (old firmware; slow, aged, or failing drives)
+
+   Prioritize upgrading these areas before focusing on compute-related performance constraints.
+   
+   For example:
+
+   - 1GbE network link can support approximately 125MB/s of throughput - which may not support the full capabilities of even a single spinning disk
+   - 10GbE network can support approximately 1.25GB/s, potentially supporting 4-6 spinning disk drives running at ~200MB/s (such as Western Digital Gold drives)
+   - 25GbE network can support approximately 3.125GB/s, potentially supporting 12 disks
+
+   The recommended minimum MinIO cluster of 4 nodes with 4 drives each (16 total disks) requires a 50GbE network to support the total potential aggregate throughput.
+
+   MinIO takes full advantage of the modern hardware improvements such as AVX-512 SIMD acceleration, 100GbE networking, and NVMe SSDs when available.
+
+Recommended Hardware Tests
+--------------------------
+
+MinIO Diagnostics
+~~~~~~~~~~~~~~~~~
+
+Run the following command.
+If you have access to :ref:`SUBNET <minio-docs-subnet>`, you can upload the results there.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support diag ALIAS
+
+Drive Tests
+~~~~~~~~~~~
+
+Test each drive independently on all servers to ensure they are identical in performance.
+Use the results of these OS-level tools to verify the capabilities of your storage hardware.
+Record the results for later reference.
+
+#. Test the drive's performance during write operations
+
+   This tests checks a drive's ability to write new data (uncached) to disk by creating a specified number of blocks at up to a certain number of bytes at a time to mimic how a drive would function with writing uncached data. 
+   This allows you to see the actual drive performance with consistent file I/O.
+   
+   .. code-block::
+      :class: copyable
+
+      dd if=/dev/zero of=/mnt/driveN/testfile bs=128k count=80000 oflag=direct conv=fdatasync > dd-write-drive1.txt
```

Comment:
Drive testing outside of MinIO can for the most part be replaced with `mc support {drive,net}`. If those results come back unexpected (perf way outside of expected range) then users can fall back to dd/fio but otherwise it is a heavy lift now that we have tooling included in the product.

Reply:
I've created sections for mc support tests, default OS tests, and 3rd party tests.